TLM/TLM/Constants.cs

```diff
@@ -40,5 +40,8 @@ public static class Constants {
         }
 
         public static IManagerFactory ManagerFactory => Manager.Impl.ManagerFactory.Instance;
+
+        public  const float MAX_HIT_ERROR = 2.5f;
```

Comment:
Add a comment please what is this (maybe also where this is to be used) and the unit (metres, probably)